mkdocs.yml

```diff
@@ -97,6 +97,7 @@ nav:
             - Additional Details:
               - Device Profile: microservices/core/metadata/details/DeviceProfile.md
               - Device System Events: microservices/core/metadata/details/DeviceSystemEvents.md
+              - Special Character: microservices/core/metadata/details/SpecialChar.md
```

Comment:
move to under `Cross Cutting Concerns` because it can be applied to all other services